src/Xhgui/Db/Mapper.php

```diff
@@ -57,10 +57,10 @@ protected function _conditions($search)
             $conditions['meta.simple_url'] = (string)$search['simple_url'];
         }
         if (!empty($search['request_start'])) {
-            $conditions['meta.SERVER.REQUEST_TIME']['$gte'] = $this->_convertDate($search['request_start']);
+            $conditions['meta.profile_ts']['$gte'] = $this->_convertDate($search['request_start']);
```

Comment:
This will break existing installs as old data will not have this key. Can we add the new key in a more compatible way?